rerun_py/src/python_bridge.rs

```diff
@@ -238,6 +238,18 @@ fn new_recording(
         default_store_id(py, StoreKind::Recording, &application_id)
     };
 
+    // NOTE: The Rust-side of the bindings must be in control of the lifetimes of the recordings!
+    // Check if recording already exists first.
+    // Put it in this scop to drop the mutex as early as possible as all_recordings function returns a mutex guard, and will be called again at the end of this function.
+    if let Some(existing_recording) = all_recordings().get(&recording_id) {
+        let _=utils::py_log_warn(
+            format!("Recording with id: {} already exists, will ignore creation and return existing recording.",
+            &recording_id).as_str()
+        );
+        // py_rerun_warn(c"Recording Stream already exists, will ignore creation and return existing recording.");
```

Comment:
redundant?
```suggestion

```

Reply:
True, sorry missed removing the comment.